src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java

```diff
@@ -122,11 +122,11 @@ private String getResultIcon(Result result) {
         if (result == Result.SUCCESS) {
             return ":white_check_mark:";
         } else if (result == Result.ABORTED) {
-            return ":point_up:";
+            return "";
```

Comment:
Maybe we could use the [octagonal sign](http://emojipedia.org/octagonal-sign/)? That would be `:octagonal_sign:` here.

Only downside is that neither Slack nor Mattermost support it yet.

Reply:
That's already used for "Unstable"...

Reply:
How about [Raised Hand :hand:](https://emojipedia.org/raised-hand/) `:hand:`? It's in Unicode 6.0 from 2010 and also available in Mattermost and Slack.

Reply:
What's the point of having an icon in the first place, when different statuses map to the same icon?

Reply:
The icon can signify the severity, not just individual status. Ideally we would have different icons for different statuses but if that's not possible because nothing fits "aborted" and is widely supported by all apps and browsers then it's not a bad compromise to let the icon to signify the severity instead.

Reply:
AFAICT a concept of "severity" does not exist in Jenkins. I also think it's questionable wheter a build which has completed (albeit unstable) should be the same "severity" as a build which hasn't completed at all (because it was aborted).

> nothing fits "aborted"

That is highly subjective. Even the Emojipedia page states that :hand: can mean "Stop"…

Trying to be constructive, though, here's another suggestion: :no_entry_sign: `:no_entry_sign:`

Reply:
This is an interesting problem, seems like we need a UX person to comment! ;)

✋  can mean stop but it also kind of looks like a high five to me.

In Jenkins itself, colored balls differentiate the build result. Blue for success, red for failure, yellow for "unstable" (used by some plugins and other custom cases, site-dependent), and grey for aborted.